### PR TITLE
Add Vuetify test to "popular libraries" examples

### PR DESCRIPTION
**Pull request by bennettdams**
We have a test example for Vuetify, which could be shown in the list of examples for popular libraries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +127,7 @@ Some included are:
 - [`vue-router`][vue-router-example]
 - [`vee-validate`][vee-validate-example]
 - [`vue-i18n`][vue-i18n-example]
+- [`vuetify`][vuetify]
```
Comment (afontcu) [inline README.md]:
```suggestion
- [`vuetify`][vuetify-example]
```

File: README.md [hunk 2]
```diff
@@ -194,4 +195,5 @@ If you want to lint test files that use Vue Testing Library, you can use the off
 [vue-router-example]: https://github.com/testing-library/vue-testing-library/blob/master/src/__tests__/vue-router.js
 [vee-validate-example]: https://github.com/testing-library/vue-testing-library/blob/master/src/__tests__/validate-plugin.js
 [vue-i18n-example]: https://github.com/testing-library/vue-testing-library/blob/master/src/__tests__/vueI18n.js
+[vuetify]: https://github.com/testing-library/vue-testing-library/blob/master/src/__tests__/vuetify.js
```
Comment (afontcu) [inline README.md]:
```suggestion
[vuetify-example]: https://github.com/testing-library/vue-testing-library/blob/master/src/__tests__/vuetify.js
```

**Comment by bennettdams** [comment]
Thanks, updated the link names.

**Pull request merged**